code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -954,6 +954,11 @@
 	gib()
 	return
 
+/mob/living/silicon/robot/update_canmove() // to fix lockdown issues w/ chairs
+    if(lockcharge)
+        canmove = 0

```

Comment:
It is; line 819 in `code/modules/mob/mob.dm`. `canmove = !incapacitated(INCAPACITATION_DISABLED)`
If the cyborg is not disabled, its canmove will be set to 1.